enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,1319 @@
+     service.beta.kubernetes.io/aws-load-balancer-healthcheck-healthy-threshold: "2"
+     service.beta.kubernetes.io/aws-load-balancer-healthcheck-interval: "10"
+     service.beta.kubernetes.io/aws-load-balancer-healthcheck-timeout: "4"
+     service.beta.kubernetes.io/aws-load-balancer-healthcheck-unhealthy-threshold: "2"
+     service.beta.kubernetes.io/aws-load-balancer-type: nlb
+     traffic-policy.network.alpha.openshift.io/local-with-fallback: ""
+   creationTimestamp: "2024-04-10T00:20:17Z"
+   finalizers:
+     - service.kubernetes.io/load-balancer-cleanup
+   labels:
+     app: router
+     ingresscontroller.operator.openshift.io/owning-ingresscontroller: test
+     router: router-test
+   name: router-test
+   namespace: openshift-ingress
+   ownerReferences:
+     - apiVersion: apps/v1
+       controller: true
+       kind: Deployment
+       name: router-test
+       uid: 466ac236-050b-4583-b2d1-99323f31fcba
+   resourceVersion: "57990"
+   uid: 812fb12c-8932-44bc-87fd-c257b7c0a455
+ spec:
+   allocateLoadBalancerNodePorts: true
+   clusterIP: 172.30.242.110
+   clusterIPs:
+     - 172.30.242.110
+   externalTrafficPolicy: Local
+   healthCheckNodePort: 31251
+   internalTrafficPolicy: Cluster
+   ipFamilies:
+     - IPv4
+   ipFamilyPolicy: SingleStack
+   ports:
+     - name: http
+       nodePort: 30920
+       port: 80
+       protocol: TCP
+       targetPort: http
+     - name: https
+       nodePort: 30575
+       port: 443
+       protocol: TCP
+       targetPort: https
+   selector:
+     ingresscontroller.operator.openshift.io/deployment-ingresscontroller: test
+   sessionAffinity: None
+   type: LoadBalancer
+ status:
+   loadBalancer:
+     ingress:
+       - hostname: a812fb12c893244bc87fdc257b7c0a45-e77e94821922b90d.elb.us-east-1.amazonaws.com
+```
+
+```sh
+ % oc get pods -n openshift-ingress
+NAME                             READY   STATUS    RESTARTS   AGE
+router-default-7b4f96d59-qfj7z   1/1     Running   0          112m
+router-default-7b4f96d59-tthts   1/1     Running   0          112m
+router-test-5bbb685585-2g5f7     1/1     Running   0          7m38s
+router-test-5bbb685585-np9w4     1/1     Running   0          7m38s
+```
+
+#### Cluster administrator updates eipAllocations field of the existing ingress controller
+
+AWS platform requires deleting and recreating a load balancer to change its EIPs. 
+This operation is disruptive: It interrupts ingress traffic and may cause the load balancer's address
+to change. On these platforms, the operator signals that the user must delete the Kubernetes Service of type `LoadBalancer` object.
+Once the user performs this step, the operator recreates the load balancer with the desired scope to complete the operation.
+
+In order to signal that the user must take action, the operator sets the Progressing status condition to True with a message that
+indicates the action that the user must take:
+
+```yaml
```

Comment:
I would replace all of this with just the message: 

'One or more managed resources are progressing: The IngressController
      scope was changed from "eipalloc-0956fea34de4cb7ab,eipalloc-0e9a3077a70de050a,eipalloc-0b69fc4691f54cdd0,eipalloc-01e6ba6cbba1a391b,eipalloc-0e242df173f906112"
      to "eipalloc-0387f99f5d4724c3e,eipalloc-0b09650c180c2abb6,eipalloc-0161deab2f05fe2fe,eipalloc-0ec5738e0e3808b8a,eipalloc-09d56b78479ac651d".  To
      effectuate this change, you must delete the service: `oc -n openshift-ingress
      delete svc/router-test`; the service load-balancer will then be deprovisioned
      and a new one created.  This will most likely cause the new load-balancer to
      have a different host name and IP address from the old one''s.  Alternatively,
      you can revert the change to the IngressController: `oc -n openshift-ingress-operator
      patch ingresscontrollers/test --type=merge --patch=''{"spec":{"endpointPublishingStrategy":{"loadBalancer":{"providerParameters":{"type":"AWS","aws":{"type":"NLB","eipAllocations":["eipalloc-0956fea34de4cb7ab","eipalloc-0e9a3077a70de050a","eipalloc-0b69fc4691f54cdd0","eipalloc-01e6ba6cbba1a391b","eipalloc-0e242df173f906112"]}}}}}}'''

Reply:
I have kept some pieces which show status and eipAllocations